src/bsoncxx/private/itoa.cpp

```diff
@@ -1054,29 +1060,5 @@ void itoa::init() {
     }
 }
 
-itoa::itoa() : itoa(0) {}
-
-itoa::itoa(const itoa& rhs) : itoa(rhs._val) {}
-
-itoa& itoa::operator=(const itoa& rhs) {
```

Comment:
I removed the copy and assignment operators since coverity was warning about not checking for self-assign and I didn't want to deal with it. The only case we actually need to support is assigning an integer to it.